helm/values.yaml

```diff
@@ -23,7 +24,7 @@ container:
 ingress:
   serviceName: mine-support
   endPoint: mine-support
-  server: defradev.com
+  server: vividcloudsolutions.co.uk
```

Comment:
Should this value be part of the production values file?  Are we hosting the PR deployments under this domain too?

Reply:
We only have one temporary domain at the moment and it's pointing at the IP address of the mine-support cluster so both PRs and 'releases' are hosted on the same address. 
It makes sense to have it as the default for now until we have others set up. 

The server can be overwritten by setting the server value in the release pipeline so the domain can be changed without code changes.